Survey/Survey/Network/MockServer.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import Combine
+class MockServer {
+    func loadInquiries() -> AnyPublisher<[Inquiry], Error> {
+        guard let url = Bundle.main.url(forResource: "db", withExtension: "json") else {
+            let error = NSError(domain: "", code: -1, userInfo: [NSLocalizedDescriptionKey: "File not found"])
+            return Fail(error: error).eraseToAnyPublisher()
+        }
+        return URLSession.shared.dataTaskPublisher(for: url)
```

Comment:
Can you help explain exactly what this code is doing?

Reply:
If the file is not found, it creates an error and returns a failed publisher (Fail) that emits this error. It tells you right away that there’s an error, if there is one.  It doesn’t send any values; it just completes with an error.

Reply:
Cool yeah that makes sense. Nice work!